modules/lib/default.nix

```diff
@@ -9,6 +9,7 @@ rec {
   generators = import ./generators.nix { inherit lib; };
   gvariant = import ./gvariant.nix { inherit lib; };
   maintainers = import ./maintainers.nix;
+  nushell = import ./nushell.nix { inherit lib; };
```

Comment:
Nit: maybe put it below to the other shells (i.e. rather semantic ordering than lexicographic, as this is not really ordered lexicographically anyways)

Reply:
Done! I like it better now